mlflow/ml_package_versions.py

```diff
@@ -246,16 +246,16 @@
         },
         "models": {
             "minimum": "0.0.169",
-            "maximum": "0.0.169"
+            "maximum": "0.0.190"
         }
     },
-    "sentence-transformers": {
+    "sentence_transformers": {
         "package_info": {
             "pip_release": "sentence-transformers"
         },
         "models": {
             "minimum": "2.2.2",
-            "maximum": "2.2.2"
+            "maximum": "4.29.2"
```

Comment:
This change looks strange.

Reply:
Filed PRs to fix this:

- https://github.com/mlflow/mlflow/pull/8617
- https://github.com/mlflow/mlflow/pull/8615